src/components/app/Home.js

```diff
@@ -178,115 +189,123 @@ class Home extends PureComponent {
 
   _renderInstallInstructions() {
     return (
-      <div className="homeInstructions" key={0}>
-        <div className="homeInstructionsLeft">
-          <div style={{ textAlign: 'center' }}>
-            <img className="homeSectionScreenshot" src={PerfScreenshot} />
+      <InstructionTransition key={0}>
```

Comment:
I only added `InstructionTransition` as parent in every function. It needs to be here because it needs to hold the `key` prop.